spec/factories/helper/factory_sequence.rb

```diff
@@ -15,3 +15,9 @@
     DateTime.now + n.days
   end
 end
+
+FactoryGirl.define do
+  sequence :published_at do |n|
+    DateTime.now - n.days
```

Comment:
Do not use `DateTime.now` without zone. Use one of `Time.zone.now`, `DateTime.now.current`, `DateTime.now.in_time_zone`, `DateTime.now.utc`, `DateTime.now.getlocal`, `DateTime.now.iso8601`, `DateTime.now.jisx0301`, `DateTime.now.rfc3339`, `DateTime.now.to_i`, `DateTime.now.to_f` instead.